src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -292,23 +294,11 @@ CHIP_ERROR BLEManagerImpl::StartAdvertising(void)
     ReturnErrorOnFailure(PrepareC3CharData());
 #endif
 
-    if (!isAdvertisingRerun)
-    {
-#if CONFIG_BT_PRIVACY
-        static_assert((CHIP_DEVICE_CONFIG_BLE_ADVERTISING_TIMEOUT / 1000) <= CONFIG_BT_RPA_TIMEOUT,
-                      "BLE advertising timeout is too long relative to RPA timeout");
-        // Generate new private BLE address
-        bt_le_oob bleOobInfo;
-        err = bt_le_oob_get_local(advParams.id, &bleOobInfo);
-        VerifyOrReturnError(err == 0, MapErrorZephyr(err));
-#endif // CONFIG_BT_PRIVACY
-    }
```

Comment:
How is removal of this block connected with the device name bits?

Reply:
It's not. I just noticed that we have some leftovers in the code which are no longer needed.

Reply:
As we no longer use CONFIG_BT_PRIVACY